src/compiler/index.ts

```diff
@@ -68,14 +69,16 @@ async function azle() {
         async () => {
             createAzleDirectories();
 
-            prepareDockerImage(
-                stdioType,
-                dockerImageName,
-                dockerImagePathTar,
-                dockerImagePathTarGz,
-                dockerContainerName,
-                wasmedgeQuickJsPath
-            );
+            if (nativeCompilation === false) {
```

Comment:
Who's responsibility is it to prove that this and the actual compilation are the only things that need to behave differently because of the the nativeCompilation flag? Yours as the developer or mine as the reviewer?

Reply:
Maybe I don't understand, let's discuss